hypothesis-python/docs/tutorial/replaying-failures.rst

```diff
@@ -79,22 +81,38 @@ Hypothesis will print ``Falsifying explicit example: test_something_with_integer
 Prefer |@example| over the database for correctness
 ---------------------------------------------------
 
-While the database is useful for local development, we don't recommend relying on it for the correctness of your tests. If a test fails with a particularly interesting input, and you want to make sure your test works for this input in the future, we recommend explicitly adding it to your test with |@example|.
-
-The reason is that entries in the database may be invalidated at any time by Hypothesis, because the internal format is not necessarily stable across versions. Additionally, the database is generally not checked into version control, so failures are not shared between developers, unlike with an explicit |@example|.
+While the database is useful for quick local iteration, Hypothesis may invalidate it when upgrading (because e.g. the internal format may have changed). We therefore recommend against relying on the database for the correctness of your tests. If you want to ensure an input is run every time, use |@example|.
```

Comment:
I think that we should also note that changing your test function will usually change the database key (except for comments, decorators, and blank lines), which is another reason to prefer `@example()`